src/components/FilterBar.test.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from '@testing-library/react';
+import queryString from 'query-string';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import FilterBar from './FilterBar';
+
+const renderWithTheme = (ui) =>
+  render(<ThemeProvider theme={createTheme()}>{ui}</ThemeProvider>);
+
+const queryOptions = {
+  arrayFormat: 'bracket-separator',
+  arrayFormatSeparator: ',',
+};
+
+const params = queryString.parse(
+  { sortBy: 'amount.asc' },
+  {
+    parseNumbers: true,
+    ...queryOptions,
+  }
+);
```

Comment:
you can just set `params` to an object that contains `{sortBy: 'amount.asc'}`
queryString is probably not needed.